### ISSUE TITLE
When logged in, the items in the user menu are broken

### ISSUE THREAD

**Issue by nickserv** [toroidal-code/codeflash#226]
1. Log in.
2. Click where your username is on the upper right.
3. Click anything in the menu.
4. The page's URL will update, but the page contents will never change and nothing will load.

So far, I have only noticed this issue in production/staging.

**Comment by kristenmills** [comment]
This is a false statement. The two edit pages load eventually. The view profile generally 504s.

**Comment by nickserv** [comment]
Did you try it on production/staging? I didn't get the same problem locally.

**Comment by kristenmills** [comment]
Yes. It's the same issue that we have with the solutions page that it just takes a long time to load.
